man/systemd-keyutil.xml

```diff
@@ -64,6 +64,16 @@
 
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><command>pkcs7</command></term>
```

Comment:
hmm, maybe rename this "make-pkcs7"? this is otherwise so vague.

i mean, I don#t think we need to call it "make-pkcs7-from-pkcs1" or so, because this leaves the door nicely open to support other inputs. But I think we should really clarify that we make something here, not consume or verify or whatever...

we call this commands usually "verbs", hence there's a strong reason to actually include a verb in them...